src/main/java/core/mate/academy/model/Bulldozer.java

```diff
@@ -5,11 +5,21 @@
  * Do not remove no-args constructor
  */
 public class Bulldozer extends Machine {
+    private String buldozerField;
```

Comment:
There is a typo in the field name `buldozerField`. It should be `bulldozerField` to match the class name and maintain consistency.